test/jdk/sun/security/pkcs11/Config/ReadConfInUTF16Env.java

```diff
@@ -47,8 +48,7 @@ static class TestSunPKCS11Provider {
         public static void main(String[] args) throws Exception {
             Provider p = Security.getProvider("SunPKCS11");
             if (p == null) {
-                System.out.println("Skipping test - no PKCS11 provider available");
-                return;
+                throw new SkippedException("Skipping test - no PKCS11 provider available");
```

Comment:
minor wording nit: you don't need to say "skipping test" in the message of a SkippedException.